pandas/tests/groupby/test_categorical.py

```diff
@@ -1312,13 +1311,34 @@ def test_series_groupby_on_2_categoricals_unobserved(
         ("size", 0),
         ("skew", np.NaN),
         ("std", np.NaN),
-        ("sum", np.NaN),
+        ("sum", 0),
         ("var", np.NaN),
-    ],
+    ]
 )
-def test_series_groupby_on_2_categoricals_unobserved_zeroes_or_nans(func, zero_or_nan):
+
+
+def test_series_groupby_on_2_categoricals_unobserved_zeroes_or_nans(
+    reduction_func: str, request
+):
     # GH 17605
     # Tests whether the unobserved categories in the result contain 0 or NaN
+
+    if reduction_func == "ngroup":
+        pytest.skip("ngroup is not truly a reduction")
+
+    if reduction_func == "corrwith":  # GH 32293
```

Comment:
make sure the xfail reasons have the issue number as well

Reply:
Done